src_base/xeve_itdq.h

```diff
@@ -47,7 +47,7 @@
 void xeve_sub_block_itdq(s16 coef[N_C][MAX_CU_DIM], int log2_cuw, int log2_cuh, u8 qp_y, u8 qp_u, u8 qp_v, int flag[N_C], int nnz_sub[N_C][MAX_SUB_TB_NUM], int bit_depth);
 
 typedef void (*INV_TRANS)(s16 *, s16 *, int, int, int, int);
-INV_TRANS (*xeve_func_itrans)[16][5];
+static INV_TRANS (*xeve_func_itrans)[16][5];
```

Comment:
@Jamaika1 Could you provide the details of your PR specifically restate several functions with its new `static` type? My humble opinion is to revise the title of this PR more properly with several subordinate descriptions for the better readability of this PR.

Reply:
Hello. I wanted to test the new evc codec in GCC 10.2.1 under Windsows.
I used the function standard under gcc -std = gnu11 -ftree-vectorize -g0 -O3 -fPIC and couldn't compile the codec.
```
xevem_df.o:xevem_df.c:(.bss+0x0): multiple definition of `xevem_func_bl_mc_l'; xevem.o:xevem.c:(.bss+0xc8): first defined here
```

Reply:
@MoonkiHong Not all defined functions could be compiled in config. Some only worked.
```
-DWINDOWS_MUTEX_SYNC -DENC_ECU_ADAPTIVE -DMULTI_REF_ME_STEP -DENABLE_IBC_CHROMA_REFINE -DNET_AMVP -DET_ME_REFIDX1 -DATS_INTRA_FAST -DHTDF_FAST_TBL
```

Reply:
@MoonkiHong I compiled the codec for Windows. Unfortunately when producing movies the codec only creates the evc container header.

Reply:
@Jamaika1 Sounds reasonable. We will check out your PR shortly.